lib/utils/docker.coffee

```diff
@@ -48,6 +48,12 @@ exports.appendOptions = (opts) ->
 			alias: 't'
 		},
 		{
+			signature: 'arg'
+			parameter: 'arg'
+			description: 'Set a build-time variable (eg. "-e \'ARG=value\'"). Can be specified multiple times.'
+			alias: 'e'
```

Comment:
The thing with `--build-arg` is that its case is inconsistent with our other multi-word options (eg. `--dockerHost`) and the "build" part annoys me as superfluous (and it's the reason I omitted it in the first place). I'm fine overall with `--build-arg/-B` for the sake of consistency with Docker if others agree, however this highlights an issue with our use of names for options across the CLI.